flycheck.el

```diff
@@ -6972,6 +6973,59 @@ See URL `http://stylelint.io/'."
   :error-parser flycheck-parse-stylelint
   :modes (css-mode))
 
+(flycheck-def-args-var flycheck-cuda-args cuda
+  :package-version '(flycheck . "0.01"))
```

Comment:
Package version should be `"32"` here and for other options as well (look at the header of flycheck.el).